cves/CVE-2011-1184.yml

```diff
@@ -116,10 +121,15 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then you
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
-  answer:
-  date:
-  automated:
-  contest:
+  answer: |
+   The developers had found a separate weakness in the Tomcat's implementation for authentication 
+   about 3 months ago. They came out with a solution for that problem and when they 
+   went to code review the fix, they found that there could be stale nonce values, and
```

Comment:
explaining what nonce is and how it relates to this vulnerability would be helpful in the overall description.

Reply:
Good catch, it is very difficult to understand without that description.